README.md

```diff
@@ -61,17 +62,18 @@ WallpaperBuilder::new()?.set_wallapers(|i, screen| -> (String, Mode) {
 ```
 
 
- [__cargo_doc2readme_dependencies_info]: ggGkYW0BYXSEGyDwipHVMb5RGxgd3zutc1TvG3ARKV4UcQ1NGyM1aXabIPYbYXKEG-q_ifkYbykDG6jYqfB7tTwIG6Iqwk50WXPpG4-hgAqV1y5YYWSBg29tb3JlLXdhbGxwYXBlcnNlMC4xLjFvbW9yZV93YWxscGFwZXJz
+ [__cargo_doc2readme_dependencies_info]: ggGkYW0BYXSEGyDwipHVMb5RGxgd3zutc1TvG3ARKV4UcQ1NGyM1aXabIPYbYXKEGxx2SjxLDI8cG259aafMiiZ-G9AFtAbGXAr1Gxd4WYixdpQWYWSBg29tb3JlLXdhbGxwYXBlcnNlMC4xLjFvbW9yZV93YWxscGFwZXJz
  [__link0]: https://crates.io/crates/wallpaper
  [__link1]: https://crates.io/crates/wall
  [__link10]: https://docs.rs/more-wallpapers/0.1.1/more_wallpapers/?search=set_wallpapers_from_vec
- [__link11]: https://docs.rs/more-wallpapers/0.1.1/more_wallpapers/?search=set_random_wallpapers_from_vec
- [__link12]: https://docs.rs/more-wallpapers/0.1.1/more_wallpapers/struct.WallpaperBuilder.html
+ [__link11]: `set_random_wallpapers_from_vec()`
```

Comment:
This seems to happen because `set_random_wallpapers_from_vec()` is only available with with the `random` feature flag, witch can not be enabled by [doc2r](https://github.com/msrd0/cargo-doc2readme/issues/new) at the moment.

Reply:
I have open a issue for this https://github.com/msrd0/cargo-doc2readme/issues/189.

Reply:
This issue has been fixed.